src/zope/app/container/ftesting.zcml

```diff
@@ -44,4 +111,8 @@
       password="mgrpw" />
   <grant role="zope.Manager" principal="zope.mgr" />
 
+  <class class="zope.site.folder.Folder">
+	  <allow attributes="items __getitem__ get __contains__ __setitem__" />
```

Comment:
The Zope style for ZCML files is:

- child elements indented by 2 spaces
- element attributes indented by 4 spaces

e.g.

```xml
    <element
        attribute="value"
        attribute="value">

      <child ... />

    </element>
```